Makefile

```diff
@@ -117,6 +117,11 @@ generate:
 generate-clean:
 	rm -rf hack/code-gen
 
+.PHONY: cli-test
+# Build CLI for tests
+cli-test:
```

Comment:
why do we need a special build for tests? why is `make cli` not good enough?

Reply:
`make cli` also runs all of the code generation and such so it takes quite a while to run (30 seconds or so?). As well as all of the linting (personally, I like to lint before I commit but that's just me). I thought it made sense to have a separate build that is quick for testing. What do you think?

Reply:
I actually agree with you now that I think about it, everytime I am building CLI locally I hate when it stops on linting error. WDYT then about having just one build and one that is fast? If not we should at least comment why we need two.

Reply:
So it actually occurs to me that we don't use `make cli` in CI or releasing at all - if it is used, it's used only by developers while developing.

Maybe we just drop the slow one then?

Reply:
I'm gonna take your suggestion:

```
make cli
make cli-fast
```

And then we can discuss what to do about the other targets separately.